src/tips/en/take-high-res-screenshots.md

```diff
@@ -26,7 +26,8 @@ Here's a nice trick to do a similar thing in Chromium-based browsers (Chrome, Ed
 
 1. Open DevTools and go to the **Device Mode** tool.
 1. Click the **Dimensions** dropdown menu and select **Responsive**.
-1. Enter the desired width and height for your screenshot. For example `1500` and `5000`.
+1. Click **More options** (⋮) and select **Add device pixel ratio**. This will add a new DPR dropdown to the device toolbar where you can select the DPR value.
```

Comment:
How about this:

```suggestion
1. Click **More options** (`⋮`) and select **Add device pixel ratio**. This will add a new DPR dropdown to the device toolbar where you can select the DPR value.
1. In the DPR dropdown, select the highest value to take the highest resolution screenshot.
```